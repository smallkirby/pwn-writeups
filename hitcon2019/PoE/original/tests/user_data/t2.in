i 0 abcdefgh
c 3 3
p 0
d 0 8
n
i 0 ABCD
c 0 4
s 0
p 8
d 0 12
s 1
i 0 1234
d 0 4
c 0 4
p 0
d 0 4
