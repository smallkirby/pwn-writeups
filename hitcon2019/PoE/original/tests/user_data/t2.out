Done.
Done.
Done.
defabcgh
Switched to new tab 1.
Done.
Done.
Switched to tab 0.
Done.
defabcghABCD
Switched to tab 1.
Done.
1234
Done.
Done.
1234
